specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -1015,7 +1018,7 @@
     },
     "/detect": {
```

Comment:
We should also apply same description to `/detect?overload=stream`.

Reply:
Done